django_tables2/columns/base.py

```diff
@@ -188,6 +188,25 @@ def render(self, value):
         '''
         return value
 
+    def value(self, value, record, column, bound_column, bound_row, table):
```

Comment:
Can't we just do `**kwargs` here? Better not to have this explicitly defined in multiple places if not necessary.